src/components/board/Board.tsx

```diff
@@ -0,0 +1,82 @@
+import Image, { StaticImageData } from "next/image";
+import Link from "next/link";
+
+interface props {
+  profilePic: StaticImageData; // the picture of the member
```

Comment:
```suggestion
  profilePic: StaticImageData;
```

remove these comments